common/src/main/java/com/wynntils/services/map/type/ServiceKind.java

```diff
@@ -8,38 +8,40 @@
 import java.util.Arrays;
 
 public enum ServiceKind {
-    ALCHEMISM_STATION("Alchemism Station", Texture.ALCHEMIST_STATION),
-    ARMOUR_MERCHANT("Armour Merchant", Texture.ARMOR_MERCHANT),
-    ARMOURING_STATION("Armouring Station", Texture.ARMORING_STATION),
-    BLACKSMITH("Blacksmith", Texture.BLACKSMITH),
-    BOOTH_SHOP("Booth Shop", Texture.BOOTH_SHOP),
-    COOKING_STATION("Cooking Station", Texture.COOKING_STATION),
-    DUNGEON_SCROLL_MERCHANT("Dungeon Scroll Merchant", Texture.DUNGEON_SCROLL_MERCHANT),
-    EMERALD_MERCHANT("Emerald Merchant", Texture.EMERALD_MERCHANT),
-    FAST_TRAVEL("Fast Travel", Texture.FAST_TRAVEL),
-    HOUSING_BALLOON("Housing Balloon", Texture.HOUSING_BALLOON),
-    ITEM_IDENTIFIER("Item Identifier", Texture.ITEM_IDENTIFIER),
-    JEWELING_STATION("Jeweling Station", Texture.JEWELING_STATION),
-    LIQUID_MERCHANT("Liquid Merchant", Texture.LIQUID_MERCHANT),
-    PARTY_FINDER("Party Finder", Texture.PARTY_FINDER),
-    POTION_MERCHANT("Potion Merchant", Texture.POTION_MERCHANT),
-    POWDER_MASTER("Powder Master", Texture.POWDER_MASTER),
-    SCRIBING_STATION("Scribing Station", Texture.SCRIBING_STATION),
-    SCROLL_MERCHANT("Scroll Merchant", Texture.SCROLL_MERCHANT),
-    SEASKIPPER("Seaskipper", Texture.SEASKIPPER),
-    TAILORING_STATION("Tailoring Station", Texture.TAILORING_STATION),
-    TOOL_MERCHANT("Tool Merchant", Texture.TOOL_MERCHANT),
-    TRADE_MARKET("Trade Market", Texture.TRADE_MARKET),
-    WEAPON_MERCHANT("Weapon Merchant", Texture.WEAPON_MERCHANT),
-    WEAPONSMITHING_STATION("Weaponsmithing Station", Texture.WEAPONSMITHING_STATION),
-    WOODWORKING_STATION("Woodworking Station", Texture.WOODWORKING_STATION);
+    ALCHEMISM_STATION("Alchemism Station", Texture.ALCHEMIST_STATION, "profession:alchemism"),
```

Comment:
At first look it might be better to represent these values in code with some kind of actual class? That way it can be easier to work with/refactor?

But I am only a few classes in the review, so perhaps it is not needed at all...

Reply:
You mean the tag "profession:alchemism"? It is basically open for all to define their own tag; we provide some by default. My idea was to not make it too strict, but rather have it easily extendable. As long as you have like a category with an icon (and ideally a user-friendly name), you can tag your map features with that category. Or you can replace name and/or icon for that category. So it's kind of a loose coupling, that is only based on matching names.